### PR TITLE
Add possibility to close preview window with Cmd/Ctrl + W

### PR DESCRIPTION
**Pull request by AlexandreSi**
Resolves https://forum.gdevelop.io/t/command-w-on-a-mac/45352

### CODE REVIEW

**Review by ClementPasteau — APPROVED** [review]
nice!

File: newIDE/app/src/MainFrame/ElectronMainMenu.js [hunk 1]
```diff
@@ -9,18 +9,48 @@ import {
   type MainMenuEvent,
 } from './MainMenu';
 const electron = optionalRequire('electron');
+const remote = optionalRequire('@electron/remote');
+const app = remote ? remote.app : null;
 const ipcRenderer = electron ? electron.ipcRenderer : null;
 
 // Custom hook to register and deregister IPC listener
-const useIPCEventListener = (ipcEvent: MainMenuEvent, func) => {
+const useIPCEventListener = (
```
Comment (ClementPasteau) [inline newIDE/app/src/MainFrame/ElectronMainMenu.js:17]: maybe use object as a param for clarity

File: newIDE/app/src/MainFrame/ProjectTitlebar.js [hunk 2]
```diff
@@ -42,6 +42,9 @@ const ProjectTitlebar = React.memo<Props>(
     React.useEffect(
       () => {
         const title = [
+          // On desktop app, this title is used to know if the user is focused on the
+          // main window (IDE) or on another window (preview or external editor).
+          // Should this be changed, you should also change ElectronMainMenu.js
```
Comment (ClementPasteau) [inline newIDE/app/src/MainFrame/ProjectTitlebar.js:47]: maybe worth adding a similar warning on the jfxr/piskel/yarn editors?

Comment (AlexandreSi) [reply newIDE/app/src/MainFrame/ProjectTitlebar.js:47]: True, I'll do it

**Pull request merged**